extension/ext-comments.css

```diff
@@ -273,18 +272,18 @@ a:focus {
 
 /* Why does substack remove focus indicators from everything?! */
 .ext-comments .comment-editor textarea:focus {
-  border: 2px solid var(--background_pop) !important;
+  outline: 2px solid var(--background_pop) !important;
```

Comment:
This looks kinda ugly to me.

Maybe you can toggle a class on the document <body> element when the old style option is active?

Then we can distinguish styles like:

    body:substack-style .ext-comment > .. { something }
    body:classic-style .ext-comment > .. { something else }
    
(We could generalize this to more styles, although that's hopefully not necessary.)